packit_service_fedmsg/consumer.py

```diff
@@ -48,13 +48,29 @@ def __init__(self):
     @property
     def celery_app(self):
         if self._celery_app is None:
-            password = getenv("REDIS_PASSWORD", "")
-            host = getenv("REDIS_SERVICE_HOST", "redis")
-            port = getenv("REDIS_SERVICE_PORT", "6379")
-            db = getenv("REDIS_SERVICE_DB", "0")
-            redis_url = f"redis://:{password}@{host}:{port}/{db}"
-
-            self._celery_app = Celery(backend=redis_url, broker=redis_url)
+            broker_transport_options = {}
+            if getenv("AWS_ACCESS_KEY_ID") and getenv("AWS_SECRET_ACCESS_KEY"):
+                broker_url = "sqs://"
+                if not getenv("QUEUE_NAME_PREFIX") and not getenv("DEPLOYMENT"):
+                    raise ValueError("Neither QUEUE_NAME_PREFIX nor DEPLOYMENT set")
+                broker_transport_options["queue_name_prefix"] = (
+                    getenv("QUEUE_NAME_PREFIX") or f"packit-{getenv('DEPLOYMENT')}-"
```

Comment:
How about always setting `QEUE_NAME_PREFIX` instead and move this logic to the Ansible playbooks instead? This seams more like a deployment related logic here, so I would say that on the long run it would offer us more flexibility if we keep it in the playbooks (as opposed to have some part in the playbooks and some part in the code).